Sources/SwiftProtobuf/NameMap.swift

```diff
@@ -0,0 +1,155 @@
+// Sources/SwiftProtobuf/fieldNameMap.swift - Bidirectional field number/name mapping
```

Comment:
Oops, find-and-replace gone awry too.

There are significant enough changes within the file to defeat git's crappy rename detection :(